src/features/commits/DatasetCommitList.tsx

```diff
@@ -56,17 +71,19 @@ const DatasetCommits: React.FC<DatasetCommitsProps> = ({
               first={i === 0}
               last={i === 2}
             />))
-          : commits.map((logItem, i) => (
-            <DatasetCommitListItem
-              key={i}
-              loading={false}
-              logItem={logItem}
-              active={logItem.path === path}
-              // first={i === 0 && !editable} (restore when there is <NewVersionButton> at the top of the list)
-              first={i === 0}
-              last={i === (commits.length - 1)}
-            />
-          ))}
+          : <>
+            {commits.map((logItem, i) => (
+              <DatasetCommitListItem
+                key={i}
+                loading={false}
+                logItem={logItem}
+                active={logItem.path === path}
+                // first={i === 0 && !editable} (restore when there is <NewVersionButton> at the top of the list)
```

Comment:
If I'm translating correctly, `NewVersionButton` would be the `Edit` button we now have. I think we can remove this comment as it is no longer relevant.